icu4c/source/common/putil.cpp

```diff
@@ -533,6 +533,30 @@ uprv_fmin(double x, double y)
     return (x > y ? y : x);
 }
 
+#include <iostream>
```

Comment:
Why is `iostream` included here? I don't see it in the file `putil.cpp` on the master branch.

Reply:
Found where it was removed:
https://github.com/unicode-org/icu/commit/afda6f137878057910bf4a86bc45a03a73a1e35a#diff-08900170df63db033be7aa17ad18a0e1
FWIW, you probably want to cherry-pick that as well, as I don't think we want the bloated `iostream` in the library code.

Reply:
THanks. Added.

Reply:
thx!